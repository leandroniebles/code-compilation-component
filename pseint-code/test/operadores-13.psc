Algoritmo sin_titulo
	escribir 2(3)
	escribir 3 no
	escribir "a" no 3=3
	Escribir Verdadero no Falso
FinAlgoritmo
