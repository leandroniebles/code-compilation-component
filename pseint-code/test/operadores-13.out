=== Line 2: SynError 229
=== Line 3: SynError 224
=== Line 3: SynError 236
=== Line 3: SynError 54
=== Line 3: SynError 54
=== Line 4: SynError 54
=== Line 5: SynError 54
